content/en/docs/reference/fission-cli/fission_function_test.md

```diff
@@ -37,4 +37,4 @@ fission function test [flags]
 ### SEE ALSO
 
 * [fission function](/docs/reference/fission-cli/fission_function/)	 - Create, update and manage functions
-
+* [fission authentication](/docs/installation/authentication) - authenticate function calls
```

Comment:
This would be get overwritten by autogenerated docs. but good to keep it for now.